### PR TITLE
Decrease logging spam by showing walker logs temporarily in console

### PR DESCRIPTION
**Pull request by klingan**
Short Description: Show walking logs only temporarily at the bottom line to prevent a log spammed with this. 

Fixes:
- Walking logs do not spam the logfile anymore

### CODE REVIEW

**Comment by douglascamata** [comment]
@klingan this makes very hard to spot bugs like this: https://github.com/PokemonGoF/PokemonGo-Bot/issues/1715

**Comment by klingan** [comment]
Correct, seen that happen too when the distance suddenly went up again. 
But easier to spot other bugs that get hidden because of tons of walking loglines ;)

Might be a solution in the middle to add an extra logline when the bots starts moving to a new fort?

**Comment by douglascamata** [comment]
@klingan can u keep at least 5 last movements in the bottom?

**Comment by klingan** [comment]
Not sure if that's easy to make
What I did now is : 

[00:37:00] Moving to new fort: 0af0ee29e6644b25994f52dfc1bd7819.12
Moving towards fort 0af0ee29e6644b25994f52dfc1bd7819.12, 40.82m left



[00:37:00] Moving to new fort: 0af0ee29e6644b25994f52dfc1bd7819.12
[00:37:16] Arrived at pokestop.
[00:37:16] Lured pokemon at fort f9667177ec9744909531dc86f2303ae0.16

It will log the Moving to new fort whenever the fortID changes during walking

**Comment by douglascamata** [comment]
@TheSavior please review this (I think you won't like....)

**Comment by douglascamata** [comment]
@TheSavior you think we should create a Storage class for workers that need to storage some data for next tick?

File: pokemongo_bot/cell_workers/move_to_fort_worker.py [hunk 1]
```diff
@@ -31,6 +32,10 @@ def work(self):
         lat = nearest_fort['latitude']
         lng = nearest_fort['longitude']
         fortID = nearest_fort['id']
```
Comment (douglascamata) [inline pokemongo_bot/cell_workers/move_to_fort_worker.py]: please, `fort_id` instead of `forID`.

Comment (klingan) [inline pokemongo_bot/cell_workers/move_to_fort_worker.py]: Not my code but I will update it anyway :+1:

**Comment by TheSavior** [comment]
I haven't run this, so I'm surely ignorant here, but why wouldn't this have the same problem as we had with the progress bar? Where if something else happens, then it prints at the end of the previous line and we get weird log results?

**Comment by douglascamata** [comment]
@klingan upload a screenshot showing before/after your changes, please?

**Comment by klingan** [comment]
I can make a screencast if you want?

**Comment by douglascamata** [comment]
@klingan even better!

**Comment by klingan** [comment]
[http://screencast.com/t/JsLTQfXSWQ](http://screencast.com/t/JsLTQfXSWQ)

**Comment by TheSavior** [comment]
Thanks for that screencast. I think there is a lot of value right now in having these in the logs. Especially since we have been iterating so much on these walkers. I might be more up for this in the future if user's could configure which kinds of messages they want to see. I think that is out of scope for now though. 

I'm a 👎  on this for now.

**Comment by douglascamata** [comment]
@klingan I'm going to start work on event system usage for logs right now we will have properly using `logging ` for multiple level. You will be able to make the bot more silent if you want.

**Comment by TheSavior** [comment]
This might make more sense now that we have the event system. If you feel like there is still work that needs to be done, please open another PR.

**Pull request closed without merging**